frontend/app/components/Thermometer.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import styles from "./styles/Thermometer.module.scss";
+
+type Props = {
+  tasks: string;
+};
+
+const Thermometer: React.FC<Props> = ({ tasks }) => {
+  const taskStatusInPercent = { complete: 10, asPlanned: 30, unknown: 25, delayed: 30, failed: 5 };
+  const taskStatusInOrder = [
+    taskStatusInPercent.failed,
+    taskStatusInPercent.delayed,
+    taskStatusInPercent.unknown,
+    taskStatusInPercent.asPlanned,
+    taskStatusInPercent.complete,
```

Comment:
ich fand die Order mit grün unten wie in figma eigentlich schöner, bzw unbekannt oben-gegenüber